data/coalition/coalition outfits.txt

```diff
@@ -239,15 +239,15 @@ outfit "Refueling Module"
 
 outfit "Shield Refactor Module"
 	category "Systems"
-	cost 2843000
+	cost 8324000
 	thumbnail "outfit/shield refactor module"
-	"mass" 7
-	"outfit space" -7
-	"hull repair rate" -.2
+	"mass" 11
+	"outfit space" -11
```

Comment:
I don't think this is needed either. As I mentioned, spamming these and benefiting from it already requires investing a lot of space in total to just shields, and completely guts your hull repair. So, the size increase isn't needed.